client/src/sass/main.sass

```diff
@@ -196,6 +196,10 @@
   color: $link !important
   border-color: $link !important
 
+#reserve-submit-button
```

Comment:
Wouldn't this make more sense to be in the file that uses it (possibly as scoped sass)?

Reply:
I think, if possible, that "main.sass" should contain global CSS, whereas specific styles should be localized to components.

Reply:
would that actually work?  the id property is set in the parent component's template and the scoped sass would be in the child component so I don't think anything scoped would referencing the #id would have any effect.

and i don't think I put any non-scoped CSS inside a SFC because it just gets confusing if globally-scoped CSS could be hiding in any component anywhere in the app.  I figured best to keep that in one place.

Reply:
No, I don't think it would work for a child to specify the CSS for an element in the parent.

I agree that non-scoped CSS should not be in a SFC.

I did a global search for "reserve-submit-button" and found 4 matches... since duplicate IDs are a no-no, how about changing it to a class? Then I'll feel comfortable with the class in main.sass.

Reply:
the components were all mutually exclusive... they could never be rendered at the same time.  but I went ahead and did as you suggested and changed it to a class